lib/common/tests/schemas/pcmk__build_schema_xml_node_test.c

```diff
@@ -33,6 +33,7 @@ static int
 setup(void **state) {
     setenv("PCMK_schema_directory", PCMK__TEST_SCHEMA_DIR, 1);
     crm_schema_init();
+    pcmk__xml_test_setup_group(state);
```

Comment:
the explicit schema init is no longer needed

Reply:
right, which is why the test was failing (see comment in above thread). I figured I'd keep the explicit call to `crm_schema_init()` so that we don't have to worry about what is or is not in `pcmk__xml_test_setup_group()`, and the point of the `setenv()` is obvious. It is optional though